pep-0465/scan-ops.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# http://legacy.python.org/dev/peps/pep-0465/
+# https://www.python.org/dev/peps/pep-0465/
```

Comment:
Nope; I looked up the history on this and it was added during a period when the main site was hosted on the new system but some things, like the PEPs, were still on the old one and in the process of transitioning. This is simply a link to the PEP, so since I was moving the file anyway, it makes sense to use the most up to date link as of now.